keras/layers/convolutional.py

```diff
@@ -220,6 +215,8 @@ class Convolution2D(Layer):
         border_mode: 'valid' or 'same'.
         subsample: tuple of length 2. Factor by which to subsample output.
             Also called strides elsewhere.
+        filter_dilation: tuple of length 2. Factor for kernel dilation.
+            Used by dilated convolution(a.k.a. convolution with holes or atrous convolution).
```

Comment:
nice! since atrous convs are not that mainstream yet would you mind adding a reference paper reference to the docstring?